bin/oneoff/move_user_storage_ids_to_dashboard.rb

```diff
@@ -0,0 +1,69 @@
+require File.expand_path('../../../pegasus/src/env', __FILE__)
+require src_dir 'database'
+require 'optparse'
+
+# To run: ruby bin/oneoff/move_user_storage_ids_to_dashboard.rb
+# To revert: ruby bin/oneoff/move_user_storage_ids_to_dashboard.rb --revert
+
+# Parse options
+options = {
+  revert: false,
+}
+
+OptionParser.new do |opts|
+  opts.banner = <<~BANNER
+    Usage: #{File.basename(__FILE__)} [options]
+
+    This moves the user_storage_ids table to the Dashboard DB and renames it to user_project_storage_ids
```

Comment:
I just want to point out the new official name for the table will be `user_project_storage_ids`

Reply:
Not important, but I’ve always wondered why we put the name of a column in a table (`storage_ids`). The purpose of the `storage_id` integer is to uniquely identify an S3 “folder” (Object prefix) that contains all of the project “folders” for a given person. I don’t think we really need this functionality (our relational database knows which projects belong to which students), but would prefer `user_project_folders`  or `user_project_storage_folders`

I recognize my opinion is fairly unorthodox here :)